packages/server/babel.config.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
had to copy this over to root of server until we can figure out how to properly run in mono repo